### PR TITLE
Update netty version to 4.1.50.Final and exclude older versions

### PR DESCRIPTION
**Pull request by sunilcodes**

### CODE REVIEW

File: presto-server/pom.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
         <process-name>${project.artifactId}</process-name>
 
         <beanutils.version>1.9.4</beanutils.version>
-        <netty.version>3.9.8.Final</netty.version>
+        <netty.version>4.1.50.Final</netty.version>
```
Comment (itay) [inline presto-server/pom.xml:27]: So in the report, they only complained about the 4.1.29, yes, not 3.9.8?

Should we add a separate entry for `netty-all` and keep this one the same?

Comment (sunilcodes) [reply presto-server/pom.xml:27]: Yes, the CVE only impacts 4.1.x. I will change this to keep both version.

File: presto-server/pom.xml [hunk 2]
```diff
@@ -296,6 +296,12 @@
             <version>${netty.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-all</artifactId>
+            <version>4.1.50.Final</version>
```
Comment (itay) [inline presto-server/pom.xml:302]: Let's add a comment explaining why this is necessary.

File: pom.xml [hunk 3]
```diff
@@ -764,6 +764,12 @@
                 <groupId>com.facebook.drift</groupId>
                 <artifactId>drift-transport-netty</artifactId>
                 <version>${dep.drift.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>io.netty</groupId>
+                        <artifactId>netty-all</artifactId>
```
Comment (itay) [inline pom.xml:770]: Let's add a comment explaining why this is necessary.

**Pull request merged**